docs/concepts/middleware.md

```diff
@@ -0,0 +1,206 @@
+---
+id: middleware
+title: Middleware
+---
```

Comment:
Can you please check if we have similar content on this page https://github.com/zio/zio-http/blob/main/docs/dsl/middleware.md
If so, how are you dealing with it?

Reply:
The newer version is better than the previous one. So if we focus on showing their core values, without taking into the details, it makes them better. It is good to mention about why they exist, what problems they solve, and talk a little about aspect-oriented programming and cross-cutting concerns.

Sometimes showing one/two (simple) examples (without going into details) is sufficient.

Reply:
okay